### DISCUSSION TITLE
new mastodon site reported as suspicious site

### DISCUSSION THREAD

**Discussion by silvacraig** [mastodon/mastodon#18623] [Q&A]
Ok - so I got mastodon up and then picked up the relay and 2 days later - I can't login to my site with either chrome or firefox as the site has been reported as providing suspicious software or malware.

Can someone point me to an available thread where I might get some advice on what went wrong and whether just picking up the relay is enough to create this issue.

Its a bit off putting - as it was a new site with no members except myself - not a great experience, so I w'm wondering whether this has happended to others.

**Comment by silvacraig** [comment]
Just some more information - this is the reportfrom https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fmasto1.100flowers.tech%2Fabout

Current status
warning
This site is unsafe

The site https://masto1.100flowers.tech/about contains harmful content, including pages that:

    Try to trick visitors into sharing personal info or downloading software


But in that sense - its just mastodon - that's what it does. Do I need to turn off the ability for others to sign up. Or turn off the link to download the android app?

Its running from my house on a public ip from an Australian ISP, so maybe it looks a bit weird to google - not too many people do this from home, but its what should be possible.

Its got a lets-encrypt cert - via traefik and I own the domain. Is this just edge case being caught up by a too tight algorithm?

**Comment by silvacraig** [comment]
some interesting stuff in syslog - but will turn the site off for now - so its capture - its a vm on proxmox so ...

**Comment by silvacraig** [comment]
Ok, so to get rid of the suspicious phishing flag I had to register the url with Google Search console and put a google registration .txt entry in my dns - and it was quite immediately removed

**Comment by auutstudio** [comment]
Adding an update that this same issue occurred today 8 Nov 2022 when Google Safety marked our instance as Dangerous. I have now registered the domain with G Search Console and verified with a DNS TXT entry. The dangerous flag has not been removed  immediately, but we shall see.

For additional context, these are **the specific reasons** that Google claims Mastodon to be dangerous:

Detected Issues:
(!) Deceptive pages
Description: These pages attempt to trick users into doing something dangerous, such as installing unwanted software or revealing personal information. [Learn more](https://support.google.com/webmasters/answer/9044101#phising)

Sample URLs:  
   - https://postpandemicparty.org/auth/sign_in
   - https://postpandemicparty.org/web/home

**Comment by andreykurenkov** [comment]
FYI, this also happened with our new instance sigmoid.social on the day we announced it - I had to manually object to it and then it went away
